.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: CommCare Core Builds for Formplayer
+
+on:
+  push:
+    branches: [formplayer]
+  pull_request:
+    branches: [formplayer]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: 7.3
+          arguments: build
+      - name: Cli Jar
+        uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: 7.3
+          arguments: cliJar
```

Comment:
this won't run the tests so it will only check that the code compiles right?

Reply:
the `build` step above compiles the code and runs the tests as well. The `cliJar` step just makes the `jar` using compiled classes.

Reply:
ah, right, I missed that step.